src/sfcphys.F

```diff
@@ -548,7 +548,7 @@ subroutine sfcdiags(tsk,thflux,qvflux,cd,ch,cq,u1,v1,s1,wspd,        &
 
 
       !$acc declare &
-      !$acc present(psfc,tsk,th0,tha,qa,tsk,znt,za) &
+      !$acc present(psfc,tsk,th0,tha,qa,znt,za) &
```

Comment:
interesting. Does `tsk` ever exist on the device? Why nvhpc compiler dose not catch this error?

Reply:
@sjsprecious the syntax error came from the fact that the 'tsk' variable is included twice in the declare present clause.

Reply:
Oh, I see. Thanks for pointing it out.